packages/gestalt/src/throttle.js

```diff
@@ -9,11 +9,11 @@ export default function throttle(
   fn: (...args: *) => void,
   threshhold: number = 100
 ) {
-  let last: number;
-  let deferTimer: TimeoutID;
+  let last: number | void;
+  let deferTimer: TimeoutID | void;
```

Comment:
Is this correct? `TimeoutID` should be complete or else be nullable.

Reply:
yup, right now when we do `let deferTimer;` like that it is initialized to `undefined` until we do `deferTimer = setTimeout(...` down below in which case it is a `TimeoutID`. If we switch to `let deferTimer: ?TimeoutID = null;` then it will be nullable like you mention, do you prefer the `null` default? I can make the code change @chrislloyd since its the same effect
![image](https://user-images.githubusercontent.com/7877010/43278634-8409bd7e-90c0-11e8-9a38-9286bc69de2a.png)